crates/ruff_python_ast/src/expression.rs

```diff
@@ -393,3 +393,41 @@ impl LiteralExpressionRef<'_> {
         }
     }
 }
+
+/// An enum that holds a reference to a string-like literal from the AST.
+/// This includes string literals, bytes literals, and the literal parts of
+/// f-strings.
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum StringLike<'a> {
```

Comment:
Should `StringLike` have an `as_str()` method for easy comparision?

Reply:
I don't think so as it also contains bytes literals and implicitly concatenated strings which can possibly do an allocation.